src/sider/arbeidsuforhet/OppfyltForm.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import { BodyShort, Box, Button, Heading, Textarea } from "@navikt/ds-react";
+import { ButtonRow } from "@/components/Layout";
+import { SkjemaInnsendingFeil } from "@/components/SkjemaInnsendingFeil";
+import { Forhandsvisning } from "@/components/Forhandsvisning";
+import { useForm } from "react-hook-form";
+import {
+  VurderingRequestDTO,
+  VurderingType,
+} from "@/data/arbeidsuforhet/arbeidsuforhetTypes";
+import { useArbeidsuforhetVurderingDocument } from "@/hooks/arbeidsuforhet/useArbeidsuforhetVurderingDocument";
+import { useSendVurderingArbeidsuforhet } from "@/data/arbeidsuforhet/useSendVurderingArbeidsuforhet";
+
+const texts = {
+  title: "Skriv en kort begrunnelse for hvorfor bruker oppfyller § 8-4",
+  info: "Det du skriver her blir liggende i historikken på brukeren og bruker vil få innsyn i notatet om de har behov.",
+  begrunnelseLabel: "Begrunnelse (obligatorisk)",
+  begrunnelseDescription: "Åpne forhåndsvisning for å se innstillingen.",
+  forDuGarVidere: {
+    head: "Før du går videre",
+    step1: "Å informere bruker om utfallet i saken!",
+    step2: "Informere NAY om det var NAY som ba deg gjøre denne vurderingen.",
+  },
```

Comment:
Jeg tror Peter skriblet ned dette mens han satt og live-endret ting med oss forrige. Kan hende han tenkte å komme tilbake til det for å finskrive. Jeg synes kanskje ordlyden er litt rar og ser litt lite sammenhengende ut? 
<img width="389" alt="image" src="https://github.com/navikt/syfomodiaperson/assets/37441744/fe3546cd-dae3-4427-9504-e0ded268387d">

Forslag om noe sånt?:
**Før du går videre bør du gjøre følgende**
1. Informer bruker om utfallet av vurderingen
2. Informer NAV Arbeid og ytelser via Gosys dersom det var de som initierte vurderingen av arbeidsuførheten

(For noen er det kanskje mer naturlig å gjøre dette _etter_ vurderingen?)

Reply:
Sikkert greit det! 👍 Men jeg ville tenkt at det er greiere med infinitiv enn imperativ? Siden vi lister opp handlinger som kan gjøres, ikke beordrer veilederen til å gjøre noe 🤔

Reply:
Haha ja good point!